представления/вован-зт/выскакивать/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>popup</title>
+</head>
+<body>
+   
+    <nav class="navigation">
+        <ul class="navigation__list">
+            <li class="navigation__item">
+                <a href="#"><img class="navigation__image" src="icons/bell.png" alt="bell1"> </a>
+            </li>
+            <li class="navigation__item">
+                <a href="#"><img class="navigation__image" src="icons/bell.png" alt="bell2"></a>
+            </li>
+            <li class="navigation__item navigation__item-modal">
+                <input  class="navigation__check_input" type="checkbox" name="modal" id="modal">
+                <label  for="modal" ><img class="navigation__image" src="icons/popup-button.png" alt="popup"></label> 
+
+                <div class="modal">
+
+                    <input  class="navigation__check_input" type="checkbox" name="modal2" id="modal2">
+                    <label  for="modal2" class="modal__button"></label>
+
+                    <ul class="modal__list">
+                        <li class="modal__item"><a class="modal__link" href="#"> <img class="modal__image" src="icons/apps.png" alt=""><span class="modal__descr">item1</span></a></li>
+                        <li class="modal__item"><a class="modal__link" href="#"> <img class="modal__image" src="icons/blogs.png" alt=""><span class="modal__descr">item2</span></a></li>
+                        <li class="modal__item"><a class="modal__link" href="#"> <img class="modal__image" src="icons/calendar.png" alt=""><span class="modal__descr">item3</span></a></li>
+                        <li class="modal__item"><a class="modal__link" href="#"> <img class="modal__image" src="icons/chat.png" alt=""><span class="modal__descr">item4</span></a></li>
```

Comment:
Fix the aspect ratio of images. they look ugly( [Read this article](https://jakearchibald.com/2022/img-aspect-ratio/)
![image](https://user-images.githubusercontent.com/16196199/185201720-f078d615-4b77-4a6f-82ce-5f68d5d36967.png)